### ISSUE TITLE
Provide a way to get a list of all trove classifiers

### ISSUE THREAD

**Issue by brettcannon** [pypa/warehouse#1241]
I.e. https://pypi.python.org/pypi?%3Aaction=list_classifiers

**Comment by dstufft** [comment]
We should probably implement the old format either way since it's fairly trivial to do and I'm sure Brett isn't the only person relying on it.

**Comment by alex** [comment]
@dstufft Can you mark this with a "good for new contributor" label?

**Comment by brettcannon** [comment]
Just an FYI a JSON value would be great as that's cleaner to work with.

**Comment by dstufft** [comment]
To be clear, I think we *should* get a JSON API for it too. Sorting out a JSON endpoint for this might be a good excuse to sit down and do #284, but I think we should also implement the old format too :)

**Comment by karan** [comment]
I might be able to take this on. A couple questions before I start:

1. Would this go under `legacy`?
2. What would the end point be (same as pypi)?

**Comment by dstufft** [comment]
Yea it'd go under legacy, you could just stick it in ``warehouse/legacy/api/pypi.py``. The ``doap`` view in that same file is a good example of how to register something that uses the ``?:action=`` dispatching. that should look something like:

```python
@view_config(route_name="legacy.api.pypi.list_classifiers")
def list_classifiers(request):
    pass  # TODO: Add an Implementation
```

You'll also need to add a route in ``warehouse/routes.py``, again you can use the ``doap`` as an example, and it should look something like:

```python
config.add_pypi_action_route(
    "legacy.api.pypi.list_classifiers",
    "list_classifiers",
    domain=warehouse,
)
```

**Comment by dstufft** [comment]
Fixed in #1243, added #1244 explicitly for the JSON side of this.
